src/utils/__tests__/utils.test.ts

```diff
@@ -19,12 +19,30 @@ describe('utils', () => {
   describe('beautifyBundleName', () => {
     it('should beautifyBundleName 1', () => {
       const name = beautifyBundleName('foo/bar/oi.js');
-      expect(name).toEqual('bar-oi');
+      expect(name).toEqual('foo-bar-oi');
     });
     it('should beautifyBundleName 2', () => {
       const name = beautifyBundleName('oi.js');
       expect(name).toEqual('oi');
     });
+    it('should beautifyBundleName 3', () => {
+      const name = beautifyBundleName('./not-found-page/not-found-page/not-found-page.module');
+      expect(name).toEqual('not-found-page');
+    });
+    it('should beautifyBundleName 4', () => {
```

Comment:
I think it should have a limit, it can't be longer than... 20 symbols?

Reply:
These files are Jest snapshot files. Automatically occurred when I added tests....

Reply:
Why is there a limit at all? I kept the default behavior but fixed the case that reduce the string to empty!

Reply:
Sorta, the change is subtle. It exceeds 20 characters, then it will use the last value in the path split. The last value in the array (even if that exceeds the limit)